### PR TITLE
WIP: Remove redundant replace directives

### PR DESCRIPTION
**Pull request by mauri870**
The replace directives for the packages `github.com/ajg/form` and `github.com/samalba/dockerclient` are both redundant.

We can require `github.com/cezarsa/form` directly and bypass the replace.

	gofmt -w -r '"github.com/ajg/form" -> "github.com/cezarsa/form"' .

The second replaced package does not seem to be in use according to `go list -m all`.

This also requires a recent enough commit of tsuru after tsuru/tsuru#2703.

There is a hidden bonus that `go install github.com/tsuru/tsuru-client/tsuru@latest` is now a thing.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -31,6 +31,7 @@ require (
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
 	github.com/Microsoft/go-winio v0.6.0 // indirect
+	github.com/ajg/form v0.0.0-20160822230020-523a5da1a92f // indirect
```
Comment (mauri870) [inline go.mod:34]: Leaving this as a reminder, before merging this MR we need a `go get github.com/tsuru/tsuru@latest` after https://github.com/tsuru/tsuru/pull/2703 is merged.

**Comment by mauri870** [comment]
This should be ready to review now @wpjunior, thanks!

**Comment by mauri870** [comment]
Updating Tsuru to the latest version introduced some CI failures that are unrelated to the changes here. I managed to get the tests to pass again, but I'm not completely sure if the changes are correct. I will leave some comments in these places in the diff.

File: tsuru/client/plan.go [hunk 2]
```diff
@@ -201,7 +206,11 @@ func renderPlansK8SFriendly(plans []apptypes.Plan, showMaxBurstAllowed bool) str
 	for _, p := range plans {
 		memory := resource.NewQuantity(p.Memory, resource.BinarySI).String()
 		cpuRequest := resource.NewMilliQuantity(int64(p.CPUMilli), resource.DecimalSI).String()
-		maxCPULimit := resource.NewMilliQuantity(int64(float64(p.CPUMilli)*p.CPUBurst.MaxAllowed), resource.DecimalSI).String()
+		maxAllowed := 0.0
+		if p.CPUBurst != nil {
+			maxAllowed = p.CPUBurst.MaxAllowed
+		}
+		maxCPULimit := resource.NewMilliQuantity(int64(float64(p.CPUMilli)*maxAllowed), resource.DecimalSI).String()
```
Comment (mauri870) [inline tsuru/client/plan.go:213]: @wpjunior I'm not sure if these changes are correct. Basically in tsuru latest `Plan.CPUBurst` and `Plan.Override` are now pointers so we have to check for a nil dereference. All the fixes here were panicking the tests, I will do a code search for the use of these types and update accordingly.

Comment (wpjunior) [reply tsuru/client/plan.go:213]: @mauri870 just a adjustment: maxAllowed default is 1.0, just this adjustment to allow this MR.

Comment (mauri870) [reply tsuru/client/plan.go:213]: Thanks, I fixed it. I'm pretty sure there are more cases where we can get panics if we do not check for nil. If you feel like this is fine you can merge this as-is and fix these later. I'd advise to search for the uses of Override and CPUBurst and add the nil checks as needed.

**Pull request merged**